package.json

```diff
@@ -9,10 +9,10 @@
     "gcs-upload": "build/src/cli.js"
   },
   "scripts": {
-    "test": "mocha build/test -r source-map-support/register",
+    "test": "mocha build/test -r source-map-support/register && gts check",
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "compile": "gts fix && tsc -p .",
```

Comment:
This I don't like. If I have an unsaved buffer in an editor, this is going to clobber. Running `gts fix` should be manual / explicit operation.

Reply:
Here I thought I was being clever.

Reply:
Fixed :)